prymer/primer3/primer3.py

```diff
@@ -371,12 +365,25 @@ def design_primers(self, design_input: Primer3Input) -> Primer3Result:  # noqa:
                 f"Error, trying to use a subprocess that has already been "
                 f"terminated, return code {self._subprocess.returncode}"
             )
-
-        design_region: Span = self._create_design_region(
-            target_region=design_input.target,
-            max_amplicon_length=design_input.params.max_amplicon_length,
-            min_primer_length=design_input.params.min_primer_length,
-        )
+        design_region: Span
+        match design_input.task:
+            case PickHybProbeOnly():
+                if design_input.target.length < design_input.probe_params.probe_sizes.min:
+                    raise ValueError(
+                        "Target region required to be at least as large as the"
+                        " minimal probe size: "
+                        f"target length: {design_input.target.length}, "
+                        f"minimal probe size: {design_input.probe_params.probe_sizes.min}"
+                    )
+                design_region = design_input.target
```

Comment:
@emmcauley This line caught my eye due to the lack of test coverage.

This looks like a functional change - I'd suggest backing this out to be introduced by another PR (e.g. #50 where `PickHybProbeOnly` is introduced).